app/src/main/java/org/fossasia/pslab/communication/sensors/HMC5883L.java

```diff
@@ -30,44 +30,44 @@
 
     private int measurementConf = 0;
 
-            //--------CONFB register bits. 0x01-----------//
+    //--------CONFB register bits. 0x01-----------//
 
     private int gainValue = 7;      //least sensitive
     private ArrayList<Integer> gainChoices = new ArrayList<>(Arrays.asList(8, 7, 6, 5, 4, 3, 2, 1));
-    private ArrayList<Double>  scaling = new ArrayList<>(Arrays.asList(1370., 1090., 820., 660., 440., 390., 330., 230.));
+    private ArrayList<Double> scaling = new ArrayList<>(Arrays.asList(1370., 1090., 820., 660., 440., 390., 330., 230.));
     private int ADDRESS = 0x1E;
     public String name = "Magnetometer";
     public int NUMPLOTS = 3;
     public String[] PLOTNAMES = {"Bx", "By", "Bz"};
-    public HashMap<String, ArrayList> params;
+    public HashMap<String, ArrayList> params = new HashMap<>();
 
     private I2C i2c;
 
     public HMC5883L(I2C i2c) throws IOException {
         this.i2c = i2c;
 
-        params.put("init", new ArrayList(Arrays.asList("Now")));
-        params.put("setSamplesToAverage", new ArrayList(Arrays.asList(samplesToAverageChoices)));
-        params.put("setDataOutputRate", new ArrayList(Arrays.asList(dataOutputRateChoices)));
-        params.put("setGain", new ArrayList(Arrays.asList(gainChoices)));
+        params.put("init", null);
```

Comment:
This params dictionary is used in python to generate GUI dynamically.
If we make sensor specific UI then we won't be needing this.